sc/src/main/java/uk/ac/ebi/atlas/experimentpage/ExperimentController.java

```diff
@@ -44,8 +48,8 @@ public String showExperimentPage(Model model,
         Experiment<Cell> experiment = experimentTrader.getExperiment(experimentAccession, accessKey);
 
         model.addAllAttributes(experimentAttributesService.getAttributes(experiment));
-
         model.addAttribute("content", GSON.toJson(experimentPageContentForExperiment(experiment, accessKey)));
+        model.addAttribute("numberOfCells", tSnePlotServiceDao.fetchCellNumberByExperimentAccession(experimentAccession));
```

Comment:
You could move this line inside the `experimentPageContentForExperiment` method, as the number of cells are also content.